[BUG] Fails build with -Werror=format-security (default on Debian/Ubuntu)

**Describe the bug**
Compiling with the CFLAG -Werror=format-security will make the build fail. There is a potential security issue if `p->cptext` contains untrusted input (I didn't check). It's probably a good idea to set this compile flag by default.

**To Reproduce**
Steps to reproduce the behavior:

```
$ cmake .
-- /home/randall/Coding/ktx/game.q3asm
-- Configuring done
-- Generating done
-- Build files have been written to: /home/randall/Coding/ktx
$ CFLAGS='-Werror=format-security' make
Scanning dependencies of target qwprogs
[  0%] Building C object CMakeFiles/qwprogs.dir/src/clan_arena.c.o
/home/randall/Coding/ktx/src/clan_arena.c: In function ‘enable_player_tracking’:
/home/randall/Coding/ktx/src/clan_arena.c:408:21: error: format not a string literal and no format arguments [-Werror=format-security]
  408 |   G_centerprint(e, e->cptext);
      |                    ~^~~~~~~~
/home/randall/Coding/ktx/src/clan_arena.c: In function ‘show_tracking_info’:
/home/randall/Coding/ktx/src/clan_arena.c:1162:22: error: format not a string literal and no format arguments [-Werror=format-security]
 1162 |    G_centerprint(p, p->cptext);
      |                     ~^~~~~~~~
/home/randall/Coding/ktx/src/clan_arena.c:1169:22: error: format not a string literal and no format arguments [-Werror=format-security]
 1169 |    G_centerprint(p, p->cptext);
      |                     ~^~~~~~~~
/home/randall/Coding/ktx/src/clan_arena.c: In function ‘CA_Frame’:
/home/randall/Coding/ktx/src/clan_arena.c:1334:24: error: format not a string literal and no format arguments [-Werror=format-security]
 1334 |      G_centerprint(p, p->cptext);
      |                       ~^~~~~~~~
/home/randall/Coding/ktx/src/clan_arena.c:1348:25: error: format not a string literal and no format arguments [-Werror=format-security]
 1348 |       G_centerprint(p, p->cptext);
      |                        ~^~~~~~~~
/home/randall/Coding/ktx/src/clan_arena.c:1358:23: error: format not a string literal and no format arguments [-Werror=format-security]
 1358 |     G_centerprint(p, p->cptext);
      |                      ~^~~~~~~~
/home/randall/Coding/ktx/src/clan_arena.c:1366:23: error: format not a string literal and no format arguments [-Werror=format-security]
 1366 |     G_centerprint(p, p->cptext);
      |                      ~^~~~~~~~
/home/randall/Coding/ktx/src/clan_arena.c:1371:23: error: format not a string literal and no format arguments [-Werror=format-security]
 1371 |     G_centerprint(p, p->cptext);
      |                      ~^~~~~~~~
cc1: some warnings being treated as errors
make[2]: *** [CMakeFiles/qwprogs.dir/build.make:459: CMakeFiles/qwprogs.dir/src/clan_arena.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1904: CMakeFiles/qwprogs.dir/all] Error 2
make: *** [Makefile:103: all] Error 2
```

**Expected behavior**
Compile finishes happily with return code of 0.

**Screenshots**
%

**Environment(please complete the following information):**
 - Operation System: Debian bullseye
 - Processor Architecture: x86_64
 - MVDSV Version: (doesn't apply)
 - KTX Version: tip of master

--- a/src/clan_arena.c
+++ b/src/clan_arena.c
@@ -405,7 +405,7 @@ void enable_player_tracking(gedict_t *e, int follow)
 
 		G_sprint(e, 2, "tracking %s\n", redtext("disabled"));
 		sprintf(e->cptext, "%s", "");
-		G_centerprint(e, e->cptext);
+		G_centerprint(e, "%s", e->cptext);
 
 		e->tracking_enabled = 0;
 		SetVector(e->s.v.velocity, 0, 0, 0);
@@ -1159,14 +1159,14 @@ void show_tracking_info(gedict_t *p)
 			sprintf(p->cptext, "\n\n\n\n\n\n%s\n\n\n%d\n\n\n seconds to respawn\n", 
 								redtext(p->track_target->netname), (int)ceil(p->seconds_to_respawn));
 
-			G_centerprint(p, p->cptext);
+			G_centerprint(p, "%s", p->cptext);
 		}
 		else
 		{
 			sprintf(p->cptext, "\n\n\n\n\n\n%s\n\n\n\n\n\n\n", 
 								redtext(p->track_target->netname));
 
-			G_centerprint(p, p->cptext);
+			G_centerprint(p, "%s", p->cptext);
 		}
 	}
 }
@@ -1331,7 +1331,7 @@ void CA_Frame(void)
 				{
 					p->spawn_delay = 0;
 					sprintf(p->cptext, "%s\n", "FIGHT!");
-					G_centerprint(p, p->cptext);
+					G_centerprint(p, "%s", p->cptext);
 					k_respawn(p, true);
 
 					// Don't take damage while respawning
@@ -1345,7 +1345,7 @@ void CA_Frame(void)
 					if (!p->tracking_enabled)
 					{
 						sprintf(p->cptext, "\n\n\n\n\n\n\n\n\n%d\n\n\n seconds to respawn\n", (int)ceil(p->seconds_to_respawn));
-						G_centerprint(p, p->cptext);
+						G_centerprint(p, "%s", p->cptext);
 					}
 				}
 			}
@@ -1355,7 +1355,7 @@ void CA_Frame(void)
 				sprintf(p->cptext, "\n\n\n\n\n\n%s\n\n\n%s\n\n\n\n", 
 								redtext("stay alive!"), last_alive == 999 ? " " : redtext(str_last_alive));
 
-				G_centerprint(p, p->cptext);
+				G_centerprint(p, "%s", p->cptext);
 			}
 			else if (p->in_play && p->alive_time > 2 && e_last_alive)
 			{
@@ -1363,12 +1363,12 @@ void CA_Frame(void)
 				sprintf(p->cptext, "\n\n\n\n\n\n%s\n\n\n%s\n\n\n\n", 
 								"one enemy left", e_last_alive == 999 ? " " : str_e_last_alive);
 
-				G_centerprint(p, p->cptext);
+				G_centerprint(p, "%s", p->cptext);
 			}
 			else if (p->in_play && p->alive_time > 2)
 			{
 				sprintf(p->cptext, " ");
-				G_centerprint(p, p->cptext);
+				G_centerprint(p, "%s", p->cptext);
 			}
 		}
 	}

